tests/unit/models/conftest.py

```diff
@@ -95,9 +95,8 @@ def _two_layer_model_fixture(request: Any) -> Callable[[TensorType], DeepGP]:
 
 
 # Teardown fixture to set keras floatx to float64 then return it to previous value at test finish
-# pytest uses yield in a funny way, so we use type ignore
-@pytest.fixture(name="keras_float")  # type: ignore
-def _keras_float() -> None:
+@pytest.fixture(name="keras_float")
```

Comment:
should we then move this fixture up to `tests/conftest.py` since we have it now in multiple places?

Reply:
:+1: but planning to tidy up the conftests in a separate PR